Libraries/CMSIS/Device/Maxim/MAX32520/Source/GCC/max32520.mk

```diff
@@ -47,9 +47,26 @@ ifeq "$(STARTUPFILE)" ""
 STARTUPFILE=startup_$(TARGET_LC).S
 endif
 
-ifeq "$(LINKERFILE)" ""
-LINKERFILE=$(CMSIS_ROOT)/Device/Maxim/$(TARGET_UC)/Source/GCC/$(TARGET_LC).ld
-endif
+ifeq "$(RISCV_CORE)" "" # RISCV
+# Default linkerfile is only specified for standard Arm-core projects.
+# Otherwise, gcc_riscv.mk sets the appropriate riscv linkerfile.
+LINKERFILE ?= $(TARGET_LC).ld
```

Comment:
Should SBT==1 check here, before setting LINKERFILE?
Otherwise it seems not works for ME10/ME14/ME12/ME18 which has secure + non-secure version MCUs.

Reply:
Since `SBT-config.mk` is included _before_ `max32xxx.mk` we shouldn't need it.  This check will be the default value if nothing has been set through the entire build process